src/databricks/labs/ucx/source_code/notebooks/commands.py

```diff
@@ -0,0 +1,48 @@
+import re
```

Comment:
why it's not in the cells.py? `%pip` is one of the line magic types.

Reply:
because there are dozens of magic lines (run `%lsmagic` in. notebook), not sure we want to pollute cells.py with all that stuff ?

Reply:
so `%lsmagic foo bar` gets translated into `get_ipython().run_line_magic('lsmagic', '', 'foo', 'bar')`. `%md` is one of the magics as well. `%scala`, `%r`, `%sql`, `%python` - they're all magics. Databricks REPL doesn't make a difference between cell magics and line magics, so keep it all in cells.py.

Reply:
done